ghmarkdown/ghmarkdown.py

```diff
@@ -171,10 +177,20 @@ def main():
                         help='silences server output and rate information')
     parser.add_argument('--serve', '-s', action='store_true',
                         help='locally serve parsed markdown')
-    parser.add_argument('--port', '-p', metavar='PORT')
+    parser.add_argument('--auto', '-a', action='store_true',
+                        help='automatically refreshes the server\'s html (default=2s)')
+    parser.add_argument('--rate', '-r', metavar='SEC',
+                        help='set the auto refresh rate in (integer) seconds', type=int)
+    parser.add_argument('--port', '-p', metavar='PORT',
+                        help='specifies what port to serve parsed markdown')
 
     args = parser.parse_args()
 
+    auto = args.auto
+
+    if args.rate:
+        auto_refresh = auto_refresh[:-4]+ str(args.rate) + '">/'
```

Comment:
Now that we have a format string, you can set auto refresh to be that string `% str(args.rate or 2)`